dateparser/date.py

```diff
@@ -14,6 +14,12 @@
 from dateparser.languages.detection import AutoDetectLanguage, ExactLanguages
 
 
+APOSTROPHE_LOOK_ALIKE_CHARS = [
+    u'\u2019', u'\u02bc', u'\u02bb', u'\u055a', u'\ua78c',
+    u'\uff07', u'\u2032', u'\u2035', u'\u02b9', u'\uff07',
```

Comment:
`\uxxxx` is also handy in comments.
I only learned about `\N{name}` recently myself: https://docs.python.org/2/reference/lexical_analysis.html#string-literals